tools/linker/MonoTouch.Tuner/ListExportedSymbols.cs

```diff
@@ -10,18 +10,26 @@
 using Xamarin.Linker;
 using Xamarin.Tuner;
 
-namespace MonoTouch.Tuner
+namespace Xamarin.Linker.Steps
 {
 	public class ListExportedSymbols : BaseStep
 	{
 		PInvokeWrapperGenerator state;
 		bool skip_sdk_assemblies;
 
+#if NET
```

Comment:
You don't need the #if around the entire property, only around the implementation:

```csharp
#if NET
    return LinkerConfiguration.GetInstance (Context).DerivedLinkContext;
#else
    return (DerivedLinkContext) Context;
#endif
```